src/control/lib/daos/pool_property.go

```diff
@@ -95,6 +95,31 @@ func PoolProperties() PoolPropertyMap {
 				valueMarshaler: numericMarshaler,
 			},
 		},
+		"svc_ops_enabled": {
+			Property: PoolProperty{
+				Number:      PoolPropertySvcOpsEnabled,
+				Description: "Metadata duplicate operations detection enabled",
+				valueHandler: func(s string) (*PoolPropertyValue, error) {
+					oeErr := errors.Errorf("invalid svc_ops_enabled value %s (valid values: 0-1)", s)
+					oeVal, err := strconv.ParseUint(s, 10, 32)
+					if err != nil {
+						return nil, oeErr
```

Comment:
would it be useful to differentiate between a parse and out of range error?

Reply:
You can add the error context with Wrap, this would enable user to determine between the two causes:
```suggestion
						return nil, errors.Wrap(err, oeErr.Error())
```
The returned error will include parse failure reason.